doc/api/net.md

```diff
@@ -544,40 +729,41 @@ If `data` is specified, it is equivalent to calling
 added: v0.9.6
 -->
 
-The string representation of the local IP address the remote client is
-connecting on. For example, if you are listening on `'0.0.0.0'` and the
-client connects on `'192.168.1.1'`, the value would be `'192.168.1.1'`.
+Provides the string representation of the local IP address the remote peer is
```

Comment:
Seems a convoluted way of saying "the local address of the socket", and in the presence of ip mapping, it might not even be the IP that the remote client connected on, and the remote might be a server if this socket came from net.connect(), and what is it when its a local domain socket?